IonicPortals/IonicPortalsTests/PortalsPluginTests.swift

```diff
@@ -0,0 +1,225 @@
+//
+//  PortalsPluginTests.swift
+//  PortalsPluginTests
+//
+//  Created by Thomas Vidas on 7/13/21.
+//
+
+import XCTest
+import IonicPortals
+import Capacitor
+import Combine
+
+class PortalsPluginTests: XCTestCase {
+    func test_topicPublisher__when_SubscriptionResults_are_published__they_are_emitted_to_downstream_subscribers() {
+        var results: [SubscriptionResult] = []
+        var cancellables = Set<AnyCancellable>()
+        let topic = "test:result"
+        
+        PortalsPlugin.publisher(for: topic)
```

Comment:
Usually I like to use a var named `sut` for 'system under test' so it's a little more obvious what is being tested (besides the method name). I'm thinking just putting a comment `// SUT` above these lines might help clarity a little